### PR TITLE
Move to copier

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,28 +35,23 @@ jobs:
...
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-        with:
-          python-version: ${{ matrix.python }}
-      - run: python -m pip install --upgrade pip
-      - run: python -m pip install -r requirements/ci.txt
-      - run: tox -e ${{ matrix.tox }}
+        os: ['ubuntu-20.04']
+        python:
+          - version: '${{needs.formatting.outputs.min_python}}'
```
Comment (jl-wynen) [inline .github/workflows/ci.yml:44]: So now this only runs with a single python version?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -57,65 +64,53 @@ jobs:
...
     steps:
       - uses: actions/download-artifact@v3
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: mamba-org/setup-micromamba@v1
         with:
-          python-version: '3.10'
-      - run: conda install -c conda-forge --yes anaconda-client
-      - run: anaconda --token ${{ secrets.ANACONDATOKEN }} upload --user scipp --label main $(ls conda-package-*/*/*.tar.bz2)
-
-  manage-versions:
```
Comment (jl-wynen) [inline .github/workflows/release.yml:85]: Do we need to manually clean up the gh-pages branch?

File: docs/developer/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+tox -e linkcheck
+```
+````
+
+````{tab-item} Manually
+
+Build the documentation using
+
+```sh
+python -m sphinx -v -b html -d build/.doctrees docs build/html
```
Comment (jl-wynen) [inline docs/developer/getting-started.md:81]: This still uses a `build` folder. But tox doesn't and the folder is no longer ignored by git.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (SimonHeybrock) [reply .github/workflows/ci.yml:44]: Yes, I considered keeping multiple versions, but felt there was little benefit at this point.

File: .github/workflows/release.yml (hunk 2 above)
Comment (SimonHeybrock) [reply .github/workflows/release.yml:85]: I think the settings in the deploy action in `docs.yml` might wipe that automatically?

Comment (jl-wynen) [reply .github/workflows/release.yml:85]: Looks like it. Let's keep an eye on it in the next release.

File: docs/developer/getting-started.md (hunk 3 above)
Comment (SimonHeybrock) [reply docs/developer/getting-started.md:81]: Good catch, I opened https://github.com/scipp/copier_template/issues/63.

**Review by jl-wynen — APPROVED** [review]
The branch protection rules need to be updated. Otherwise, lgtm

**Pull request merged**